### PR TITLE
GO-1682 Ignore errors on object list object type update

### PR DESCRIPTION
**Pull request by KirillSto**

### CODE REVIEW

File: core/object.go [hunk 1]
```diff
@@ -522,12 +523,21 @@ func (mw *Middleware) ObjectListSetObjectType(cctx context.Context, req *pb.RpcO
...
 
 	if err := mw.doBlockService(func(bs *block.Service) (err error) {
+		var (
+			mErr       multierror.Error
+			anySucceed bool
+		)
 		for _, objID := range req.ObjectIds {
 			if err = bs.SetObjectTypes(ctx, objID, []string{req.ObjectTypeId}); err != nil {
-				return err
+				mErr.Errors = append(mErr.Errors, err)
```
Comment (deff7) [inline core/object.go:532]: maybe log an error?

Comment (KirillSto) [reply core/object.go:533]: Done

**Review by deff7 — APPROVED** [review]

File: core/object.go [hunk 2]
```diff
@@ -522,12 +523,22 @@ func (mw *Middleware) ObjectListSetObjectType(cctx context.Context, req *pb.RpcO
...
 
 	if err := mw.doBlockService(func(bs *block.Service) (err error) {
+		var (
+			mErr       multierror.Error
+			anySucceed bool
+		)
 		for _, objID := range req.ObjectIds {
 			if err = bs.SetObjectTypes(ctx, objID, []string{req.ObjectTypeId}); err != nil {
-				return err
+				log.Errorf("failed to set object type to object '%s': %v", objID, err)
```
Comment (deff7) [inline core/object.go:532]: move objID to separate log field `objectID`

Comment (KirillSto) [reply core/object.go]: Thank you! Forgot about our log fields )

**Pull request merged**